flow-nodes/api-builder-plugin-format-date/src/flow-nodes.yml

```diff
@@ -11,18 +11,28 @@ flow-nodes:
         parameters:
           date:
             name: Date
-            description: The date to be formatted.
-            required: true
+            description: The date to be formatted (e.g. "2020-08-19"). Defaults to current date/time. For more information, see https://momentjs.com/docs/#/parsing/string/
```

Comment:
as mentioned you should any examples to the schema rather than the description. helps to keep things clean and consise:

This should work:
```
schema:
  type: string
  examples:
    - 2020-08-19
`